ubr/conf.py

```diff
@@ -127,5 +127,5 @@ def cfg(path, default=0xDEADBEEF):
     # https://www.postgresql.org/docs/9.2/static/libpq-pgpass.html
     #'pass':
     'host': cfg('postgresql.host', 'localhost'),
-    'port': cfg('postgresql.port', 5432),
+    'port': int(cfg('postgresql.port', 5432)),
```

Comment:
yes, I was surprised I hadn't encountered this yet. it's when you have `postgresql.port` explicitly defined in `app.cfg` that it's read in as a string. if it's not there, it uses the integer. I do the same with mysql a few lines up.